doc/glossary.rst

```diff
@@ -15,44 +15,67 @@ Frame:
   the term "frame" is used to indicate
   instances where the pose may not yet be known.
 
-Camera Pose:
-  In TeleSculptor, a camera pose
-  (often shortened to "camera")
+Camera:
+  In TeleSculptor, a camera
   refers primarily to the model
-  which describes the properties of a camera pose,
-  including attributes such as focal length
-  and world position and orientation.
+  which describes the intrinsic and extrinsic parameters of a camera
+  at a given frame.
```

Comment:
no objection , but not sure what your wrapping rule is here.  I was just wrapping at 80 characters or end of sentence.

Reply:
Right; not that. Since the line breaks don't matter, it's better to wrap in a way that minimizes the likelihood of needing to rewrap if the wording is changed. I already partly mentioned this elsewhere, but to be more complete:
- Keep lines under 80¹ characters (obviously).
- *Always* wrap after a sentence.
- Everywhere else, try to wrap where a pause "makes sense".
- Keeping line lengths similar is usually preferable, but not at the expense of the preceding rules.

For the second-to-last, after commas is usually a good place. I will sometimes break after each item in a list (should help with diff churn if an item needs to be added or removed). Otherwise, thinking about where a pause would be least awkward if the text was spoken aloud seems to be a good "rule of thumb". (I'll sometimes "fudge" this a bit for the sake of the last rule, but breaking somewhere that would clearly be an awkward verbal pause is to be avoided.)

In a sense, it's not that different from how we would break if it was code. (Break after statements, prefer to keep related things together, strive for similar line lengths but not at the expense of logical flow.)

(¹ As always, that's also pedantically "79 characters or less, unless you really, really have to have 80".)